docker/spec/dependabot/docker/update_checker_spec.rb

```diff
@@ -415,6 +415,7 @@
 
     context "when the dependency has a namespace" do
       let(:dependency_name) { "moj/ruby" }
+      let(:version) { "2.4.0" }
```

Comment:
This test was working by accident due to the downgrade behaviour the `version` value being used was `17.10`, which is the Ubuntu version set at the top level in the test that remained a mystery guest here.

This should be a 2.4.0 to 2.4.2 upgrade to align with our other Ruby specs in this file.